styles/designs/carnival/parts/lists/_lists-from-modules-shapes.scss

```diff
@@ -165,3 +165,16 @@
         )),
     )
 ));
+
+
+// Style Lists from stimulus in pdf
+
+@include create_shape('carnival:::ListsFromStimulus',(
+    _components: (
+        map-merge($ListContainer,(
+          _components: (
+            $UnorderedList--FirstLevel_FromStimulus,
+          )),
+        )
+    )
+));
```

Comment:
Could you move this to this file https://github.com/openstax/ce-styles/blob/main/styles/designs/carnival/parts/lists/_lists-from-elements-shapes.scss ? There is already shape for exercises, but for now we can't reuse it for stimulus, so lets keep a new shape. More info is in a corn: https://github.com/openstax/ce-styles/blob/31c8eddd8b577cc7887dc8ef34761626df70afb0/styles/designs/corn/parts/lists/_lists-from-elements-shapes.scss#L47

Reply:
And please remember to add word `Shape` to the name of a shape (`ListsFromStimulusShape`)

Reply:
Also, I updated the shape name, thanks for your feedback :-)